KeyValueObserving/Base.lproj/Main.storyboard

```diff
@@ -21,51 +21,51 @@
                             <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="20" translatesAutoresizingMaskIntoConstraints="NO" id="pEU-R1-q2u">
                                 <rect key="frame" x="100" y="94" width="214" height="568"/>
                                 <subviews>
-                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="inputTextLabel" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="nfH-ed-bih">
-                                        <rect key="frame" x="0.0" y="0.0" width="214" height="277.33333333333331"/>
-                                        <fontDescription key="fontDescription" type="system" pointSize="17"/>
-                                        <color key="textColor" cocoaTouchSystemColor="lightTextColor"/>
+                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="nfH-ed-bih">
+                                        <rect key="frame" x="0.0" y="0.0" width="214" height="318"/>
```

Comment:
Can you set the height to be dynamic? I don't know if that's possible on ios

Reply:
Yes, this is possible through setting constraints via Autolayout on iOS.